easybuild/easyblocks/generic/cmdcp.py

```diff
@@ -67,7 +67,9 @@ def build_step(self):
             # determine command to use
             # find (first) regex match, then complete matching command template
             cmd = None
-            for pattern, regex_cmd in self.cfg['cmds_map']:
+            for pattern, regex_cmd in self.cfg.get_ref('cmds_map'):
+                pattern = pattern % self.cfg.template_values
+                regex_cmd = regex_cmd % self.cfg.template_values
```

Comment:
Isn't that equivalent? The getter recursively resolves the templates so how is this different than before?